### DISCUSSION TITLE
Add function: Full Screen - Tile window to top and bottom

### DISCUSSION THREAD

**Discussion by jcwils** [rxhanson/RectanglePro-Community#267] [Ideas]
Is it possible to add the function...Full Screen - Tile window to top and bottom? The default MacOS behaviour is Full Screen - Tile window to left and right. What I'd like to see is 2 windows with no title bars showing and one one top of the other.

Thanks

**Comment by rxhanson** [comment]
Thanks for the feedback. Unfortunately, there's not a way to do precisely what you're looking for as a 3rd party developer. 

Version 3+ of Rectangle pro does let you resize adjacent windows when resizing a window though.
<img width="855" alt="Screenshot 2023-04-30 at 8 21 27 AM" src="https://user-images.githubusercontent.com/13651296/235352713-2cc8ef32-7e16-4837-8351-48c51589699b.png">

**Comment by jcwils** [comment]
Thanks for the quick feedback, it's a shame that Apple doesn't allow access to change the split orientation. I tried "resize adjacent windows" and it does allows me to change the split ratio.
